user/nuclei/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "nuclei"
+pkgver = "3.3.5"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["-ldflags=-s -w", f"./cmd/{pkgname}"]
```

Comment:
```suggestion
make_build_args = ["./cmd/nuclei"]
```

the strip doesn't do anything as cbuild already strips binaries